core/templates/domain/skill/SkillDifficultyObjectFactorySpec.ts

```diff
@@ -45,5 +45,25 @@ describe('Skill Difficulty object factory', () => {
       };
       expect(skillDifficulty.toBackendDict()).toEqual(skillDifficultyDict);
     });
+
+    it('should set the description of the skill difficulty instance', () => {
+      var skillDifficulty = skillDifficultyObjectFactory.create(
+        '1', 'test skill', 0.3);
+      expect(skillDifficulty.getId()).toEqual('1');
+      expect(skillDifficulty.getDescription()).toEqual('test skill');
+      expect(skillDifficulty.getDifficulty()).toEqual(0.3);
+      skillDifficulty.setDescription('new test skill');
```

Comment:
Add a new line above and below this line* (Just to highlight different section in the tests)

Reply:
Done.